runtime/compiler/optimizer/J9RecognizedCallTransformer.cpp

```diff
@@ -170,16 +172,16 @@ void J9::RecognizedCallTransformer::transform(TR::TreeTop* treetop)
    switch(node->getSymbol()->castToMethodSymbol()->getMandatoryRecognizedMethod())
       {
       case TR::sun_misc_Unsafe_getAndAddInt:
-         processUnsafeAtomicCall(treetop, node, TR::SymbolReferenceTable::atomicFetchAndAdd32BitSymbol);
+         processUnsafeAtomicCall(treetop, node, TR::SymbolReferenceTable::atomicFetchAndAddSymbol);
          break;
       case TR::sun_misc_Unsafe_getAndSetInt:
-         processUnsafeAtomicCall(treetop, node, TR::SymbolReferenceTable::atomicSwap32BitSymbol);
+         processUnsafeAtomicCall(treetop, node, TR::SymbolReferenceTable::atomicSwapSymbol);
          break;
       case TR::sun_misc_Unsafe_getAndAddLong:
-         processUnsafeAtomicCall(treetop, node, TR::SymbolReferenceTable::atomicFetchAndAdd64BitSymbol);
+         processUnsafeAtomicCall(treetop, node, TR::SymbolReferenceTable::atomicFetchAndAddSymbol);
```

Comment:
Why not common this entry with `TR::sun_misc_Unsafe_getAndAddInt`?

Reply:
Fixed in 24ee533